tools/c-parser/program_analysis.ML

```diff
@@ -1681,6 +1681,9 @@ in
       (puredecl d, pop_scope (set_proto_params fname (List.rev pm_vis0) e))
     end
   | EnumDecl eninfo => (puredecl d, process_enumdecl eninfo e)
+  (* nothing to be done *)
+  (* XXX: like process_enumdecl, use const_eval on the condition to check it? *)
+  | StaticAssertion {...} => (puredecl d, e)
```

Comment:
I guess this is the main question we have to decide on. I think we should stay with the "do nothing", but I'm keen to hear arguments for/against.